forc-plugins/forc-client/src/op/call/parser.rs

```diff
@@ -1554,4 +1656,160 @@ mod tests {
         let output = token_to_string(&token).unwrap();
         assert_eq!(output, "(container:{{42, fuel}, fuel})");
     }
+
+    #[test]
+    fn param_to_function_arg_conversion() {
+        let param_type = ParamType::Unit;
+        assert_eq!(param_to_function_arg(&param_type), "()");
+
+        let param_type = ParamType::Bool;
+        assert_eq!(param_to_function_arg(&param_type), "bool");
+
+        let param_type = ParamType::U8;
+        assert_eq!(param_to_function_arg(&param_type), "u8");
+
+        let param_type = ParamType::U16;
+        assert_eq!(param_to_function_arg(&param_type), "u16");
+
+        let param_type = ParamType::U32;
+        assert_eq!(param_to_function_arg(&param_type), "u32");
+
+        let param_type = ParamType::U64;
+        assert_eq!(param_to_function_arg(&param_type), "u64");
+
+        let param_type = ParamType::U128;
+        assert_eq!(param_to_function_arg(&param_type), "U128");
+
+        let param_type = ParamType::U256;
+        assert_eq!(param_to_function_arg(&param_type), "U256");
+
+        let param_type = ParamType::B256;
+        assert_eq!(param_to_function_arg(&param_type), "b256");
+
+        let param_type = ParamType::Bytes;
+        assert_eq!(param_to_function_arg(&param_type), "Bytes");
+
+        let param_type = ParamType::String;
+        assert_eq!(param_to_function_arg(&param_type), "str");
+
+        let param_type = ParamType::RawSlice;
+        assert_eq!(param_to_function_arg(&param_type), "RawSlice");
+
+        let param_type = ParamType::StringArray(4);
+        assert_eq!(param_to_function_arg(&param_type), "str[4]");
+
+        let param_type = ParamType::StringSlice;
+        assert_eq!(param_to_function_arg(&param_type), "str");
+
+        let param_type = ParamType::StringArray(4);
```

Comment:
`StringArray(4)` is a duplicate test

Reply:
https://github.com/FuelLabs/sway/pull/7018/commits/5c13d0701f315111b34a2e131b18fc0bf3435802